Generator creates invalid JavaScript

I'm trying to generate the code for googleapis-private/automl.  After running prettier and trying to run `npm run lint`, I get the following error:

```
beckwith-macbookpro2:nodejs-automl beckwith$ npm run lint

> automl@0.1.0 lint /Users/beckwith/Code/nodejs-automl
> repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/ smoke-test/

lint: Linting files in: /Users/beckwith/Code/nodejs-automl
lint: Running: eslint src/ samples/ system-test/ test/ smoke-test/

/Users/beckwith/Code/nodejs-automl/test/gapic-v1beta1.js
   948:11  error  'done' is already defined  no-redeclare
  1018:11  error  'done' is already defined  no-redeclare

✖ 2 problems (2 errors, 0 warnings)

lint: Oh no! Linting failed after 1.24s.
```

The code in question looks like this:

```js
it('invokes deployModel without error', done => {  
    // a bunch of code...
    var done = true;

```

This is currently blocking a launch.

--- a/src/main/java/com/google/api/codegen/util/js/JSNameFormatter.java
+++ b/src/main/java/com/google/api/codegen/util/js/JSNameFormatter.java
@@ -102,11 +102,14 @@ public class JSNameFormatter implements NameFormatter {
 
   /**
    * A set of ECMAScript 2016 reserved words. See
-   * https://tc39.github.io/ecma262/2016/#sec-reserved-words
+   * https://tc39.github.io/ecma262/2016/#sec-reserved-words A notable exception here is `done`
+   * which is not reserved but is de facto standard for mocha tests, see
+   * https://mochajs.org/#asynchronous-code for details.
    */
   public static final ImmutableSet<String> RESERVED_IDENTIFIER_SET =
       ImmutableSet.<String>builder()
           .add(
+              "done",
               "break",
               "do",
               "in",

